pathspider/cmd/measure.py

```diff
@@ -107,6 +108,10 @@ def run_measurement(args):
                 outputfile.write(json.dumps(result) + "\n")
                 logger.debug("wrote a result")
                 spider.outqueue.task_done()
+        #upload data if option is set
+        logger.info("uploading data to PTO..")
```

Comment:
Why does this log happen before the if statement below it?

Reply:
My bad